global/meta/models.yml

```diff
@@ -3825,6 +3825,9 @@ action_worker:
       - running
       - end
       - aborted
+      - warning
+      - error
+      - done
```

Comment:
Are you sure you want to add these to the general action worker? What's e.g. the difference between `end` and `done`? `warning` and `error` are really import-specific and do not convey meaning about the status of the action worker.

Reply:
I agree and we knew, that this is a misuse of the action_worker. We did it with the idea that this is the faster way to develop the imports. It should be a separate collection, but also to be written in a separate transaction in parallel to the data. We will separate action_worker and its functionality from preview_import-worker on moving to relational db-design.
The meaning of the action_worker states is not compatible with that of the preview_import.
`end` in action_worker means, that the action_worker ended and can be deleted from database. The last point is, why we didn't wanted to use this state for the preview_import. `abort` may be similar.
We could use always `running`, maybe with some statistical problems or deleting long running action workers.

`done` in preview_import means, that everything is okay without warnings (could be imported) or error (whole import may not be imported). A program, cleaning or statistic job can easily decide by the state if it is an action worker or a preview_import.

Reply:
Yes, but I don't think we should mix the stati of action workers and imports. I would suggest to leave the state of an import inside the data saved in the action worker and leave the state at running until the import is either sucessful or aborted. If we implement some cleanup tool for action workers, it can, as you said, easily decide for every action worker still running whether it is an actual action worker or an import and then decide to keep or delete it. To summarize: We already have a spearate import state inside the data, I would prefer not to mix it with the action worker's state.

Reply:
There is no separate state inside the data, there is only a row state. Lets talk about this, I'm not willing to change

Reply:
New collection import_preview introduced and not using the action_worker for this anymore.
I'll close this in favor of https://github.com/OpenSlides/openslides-backend/pull/1912